provider/contract_test.go

```diff
@@ -0,0 +1,29 @@
+package provider
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/pact-foundation/pact-go/dsl"
+	"github.com/pact-foundation/pact-go/types"
+)
+
+func TestProvider(t *testing.T) {
+	pact := &dsl.Pact{
+		Consumer:                 "ResultsRenderer",
+		Provider:                 "ResultsAPI",
+		DisableToolValidityCheck: true,
+	}
+
+	go StartProvider()
+
+	pactDir := "../consumer/pacts"
+	_, err := pact.VerifyProvider(t, types.VerifyRequest{
+		ProviderBaseURL: "http://localhost:8000",
+		PactURLs:        []string{filepath.ToSlash(fmt.Sprintf("%s/resultsrenderer-resultsapi.json", pactDir))},
```

Comment:
is `resultsrenderer-resultsapi.json` created automatically or should be included in this PR instead?

Reply:
I wouldn't add it.
We should have a script to move the resulting contract from consumer directory to provider directory.